### PR TITLE
[Auctions] Ensure that the bid state is a higher priority than regist ration pending

### PR DESCRIPTION
**Pull request by orta**
The ArtworkVC was prioritizing letting someone know that they were eligible to bid, instead of, well, letting them bid. Which you might want to know about. 👍

### CODE REVIEW

**Comment by orta** [comment]
Also removes some `will`s from the tests 🎉

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
        
            1 Warning
        
      </th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
    
    
  </tbody>
</table>
  

  



Time | Class | Function |
| --- | ----- | ----- |264.4ms | [AuctionViewController.swift](/artsy/eigen/tree/orta-auction_bid/Artsy/View_Controllers/Auction/AuctionViewController.swift#L181) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
196.7ms | [SwiftyJSON.swift](/artsy/eigen/tree/orta-auction_bid/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
175.5ms | [WebSocket.swift](/artsy/eigen/tree/orta-auction_bid/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
77.3ms | [Observable.swift](/artsy/eigen/tree/orta-auction_bid/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
74.8ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/orta-auction_bid/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
73.0ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/orta-auction_bid/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, sideMargin: String)
65.6ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/orta-auction_bid/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
63.6ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/orta-auction_bid/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)
60.9ms | [LiveAuctionLotViewController.swift](/artsy/eigen/tree/orta-auction_bid/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotViewController.swift#L47) | @objc override func viewDidLoad()
60.4ms | [RefinementOptionsViewController+Private.swift](/artsy/eigen/tree/orta-auction_bid/Artsy/View_Controllers/Auction/RefinementOptionsViewController+Private.swift#L63) | func stackView() -> ORStackView




<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

File: Artsy/Views/Styled_Subclasses/Buttons/ARBidButton.m [hunk 1]
```diff
@@ -52,9 +54,6 @@ - (void)setAuctionState:(ARAuctionState)state animated:(BOOL)animated
         enabled = YES;
         interactionEnabled = NO;
 
-    } else if (state & ARAuctionStateStarted) {
-        title = ARBidButtonBiddingOpenStateTitle;
-
```
Comment (ashfurrow) [inline Artsy/Views/Styled_Subclasses/Buttons/ARBidButton.m]: This change looks [familiar](https://github.com/artsy/eigen/pull/1665/files#diff-1a20857a03de3914ce9919f03dcd20a1R51)

screen shot 2016-06-10 at 8 37 39 am

Notice I'm also checking that the auction has started _and_ the user is registered. You reckon that's okay?

I like your changes to the tests, I think we should merge this and then I can deal with the conflict.

Comment (orta) [inline Artsy/Views/Styled_Subclasses/Buttons/ARBidButton.m]: > Notice I'm also checking that the auction has started and the user is registered. You reckon that's okay?

Yeah, probably better, as it's more specific

**Pull request merged**